concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -377,6 +377,12 @@ resources:
       region_name: ((aws_region))
       versioned_file: pagerduty-secrets.yml
 
+  - name: gh-release-noisy-neighbor-nozzle
```

Comment:
We use the git resource everywhere, why are we using the github-release here? Also with git we can set the tag as we usually do.

Reply:
We're using the github-release resource here as we can pull in the prebuilt binaries as apposed to building them ourselves.